templates/configurations/checkbox.interface.hbs

```diff
@@ -1,8 +1,8 @@
+<div v-show="source === 'defaultSource'" class="form-group">
```

Comment:
**Default values** should come after **Description**. See https://github.com/Fliplet/fliplet-studio/issues/5813#issuecomment-668048849